core/views.py

```diff
@@ -21,7 +21,7 @@ def video_upload(request):
 
 def video_detail(request, video_pk):
     video = Video.objects.get(id=video_pk)
-    return render(request, "studiopal/video_detail.html", {"video": video})
+    return render(request, "studiopal/video_detail.html", {"video":video,"video_pk":video.pk})
```

Comment:
Hey Izis! It doesn't look like `video_pk` is being used in the template, so it can be deleted.